c31-shadowlyn/react-nutshell-shadowlyn
# added edit funcitonality

# Description
Edit functionality

Applies to issue # 3

## Type of change



# Testing Instructions

1. `git fetch`
2. `git checkout sb-event-edit`
3. `npm start`
4. When a user clicks on the edit button, a form should appear with prefilled fields.
5. Hit submit, event should be updated
# Checklist:
